recipes-fsl/images/fsl-image-machine-test.bb

```diff
@@ -22,6 +22,8 @@ CORE_IMAGE_EXTRA_INSTALL += " \
     packagegroup-fsl-tools-gpu-external \
     packagegroup-fsl-tools-testapps \
     packagegroup-fsl-tools-benchmark \
+    ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', \
```

Comment:
Why are you tying it to the systemd feature?

Reply:
Because firmwared requires it.

Reply:
Humm ok.